README.md

```diff
@@ -1,27 +1,57 @@
 # mathjax-api
-A Node.js HTTP API to render MathJax equations in HTML content
 
-This service uses the MathJax-node implementation that is optimised for mathjax embedded in HTML content. It is currently limited to LaTex to SVG conversion, but feel free to extend it :)
+A Node.js HTTP API to render MathJax equations in HTML content.
 
-Defaults port 8080.
+This service uses the MathJax-node implementation that is optimised for mathjax embedded in HTML content. It is currently limited to Tex to SVG conversion.
 
-Node usage: 
-```
-$ npm install
-$ node app.js
-```
+## Getting started
 
-Test the server is working by doing a HTTP GET or navigate to http://localhost:8080
+### Prerequisites
 
-```
-$ curl http://localhost:8080
-```
+To run the API locally you must have the following:
 
-To post content to the service for conversion, put your content in the "input" field using a application/x-www-form-urlencoded format:
+1. [nvm](https://github.com/nvm-sh/nvm) installed:
 
+   ```shell
+   brew install nvm
+   ```
```

Comment:
Didn't Nick have a word with you about this? Installing NVM via brew has issues with updating.

Reply:
Will message you about this as Nick didn't mention any updating issues.